src/Main.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="pl">
+<head>
+    <title style="center">Rejestracja</title>
```

Comment:
Do elementów w <head> nie dodajemy styli, bo nie są widoczne na stronie. <title> to treść, która jest widoczna np. na zakładce przeglądarki